docs/whatsnew/2.0.rst

```diff
@@ -0,0 +1,32 @@
+.. doctest-skip-all
+
+.. _whatsnew-2.0:
+
+************************
+What's New in SunPy 2.0?
+************************
+
+Overview
+========
+
+The SunPy project is pleased to announce the 2.0 release of the sunpy package.
+
+
+.. _whatsnew-2.0-pixels:
+
+Fixes and clarification to pixel indexing
+=========================================
+
```

Comment:
we should also explicitly call out `make_fitswcs_header`

Reply:
Didn't we backport that fix? If we did it's not really a what's new in 2.0 as we already released it.